compression_test.go

```diff
@@ -42,12 +43,10 @@ func TestAcceptEncodingOrdering(t *testing.T) {
 		w.WriteHeader(http.StatusOK)
 		called = true
 	})
-	server := httptest.NewServer(verify)
-	t.Cleanup(server.Close)
-
+	server := memhttptest.NewServer(t, verify, memhttp.WithoutHTTP2())
```

Comment:
Why is this using `WithoutHTTP2` here (and throughout)? I think there are only two places in `connect_ext_test` that actually intend to force HTTP 1.1. Everything else should likely be fine using HTTP/2.

Reply:
I was keeping previous behaviour. Will change.